### DISCUSSION TITLE
Why is the my account flagged?

### DISCUSSION THREAD

**Discussion by bojanterzic529** [community/community#30093] [Other Feature Feedback, Questions, & Ideas] [answered]
Suddenly I found message like this in top of my profile page
"Your account has been flagged."
What's the matter? Can I restore this to normal state?

**Comment by flextra19** [comment, marked as answer]
No, you can't.   Creating two accounts from one device is banned.

**Reply by bojanterzic529** [reply]
ah, okay, thanks

**Reply by ghost** [reply]
I understand, thank you

**Reply by mundaneAsylum** [reply]
why is it banned if the account switcher exists?

**Reply by AsimHateez** [reply]
> No, you can't. Creating two accounts from one device is banned.

What's the solution then?

**Comment by CaptainTee** [comment]
I tried to log in to Gnosis SAFE forum with my GitHub to raise a proposal as a SAFE Guardian but it was very disappointing to see that my account has been flagged for having my profile set to hidden which I didn't even notice until I was told now.

Kindly remove the restriction on my account to enable to complete my actions, I will immediately set the profile to public as you wanted.

God bless you as you do.

**Reply by CaptainTee** [reply]
Sorry, I just read well now that my profile was hidden from the public as a result of my account being flagged.

I didn't create two accounts on same device and neither am I using two accounts. Though, I agreed a friend has once logged in her account (SpicyTee16) on my PC almost a year ago 23rd September, 2021 but never log in on my PC ever since then.


Please help me resolve this as soon as possible, God bless you as you do

**Comment by efeblissai** [comment]
So please what's now the solution

**Comment by aobapps** [comment]
Why my account is flagged? I didn't do anything wrong.

**Reply by Krishna3451** [reply]
Did you get your account back?

**Comment by Iamoptimist** [comment]
Why is my account flagged? I've done nothing wrong.

You guys have made a mistake. Kindly review it for me please

**Comment by Effiomorok** [comment]
Good day. Just saw this message pop up after I tried using the new feature of connecting more ethereum address to my gitcoin passport. How can this be resolved?

**Comment by Shamilfawas29** [comment]
What to do if your account is flagged

**Reply by iamWitchKing** [reply]
My account flagged

**Comment by ArtemKutsiy** [comment]
"Your account has been flagged" - this is what i see when i open my account! This is my first account, I created it maybe 2 weeks ago, not more than that. No additional accounts were created on my computer. I did not do any prohibited actions, except for adding my account to other sites, to participate in various sweepstakes and events. can you help me Verify information? What is the problem? Do I have to cheat the system and create another account? This is my only account and I don't need additional ones.

**Comment by Thexoxo** [comment]
hello I rarely use github and suddenly I log in to my account and my account is flagged would it be possible to know why thank you for your answer

**Comment by ghost** [comment]
Hi all!  This can happen in 3 ways(account flagged): 
1. temporary email, that is, it is not an accepted email provider like protonmail, outlook, gmail etc.
2. another problem that can occur is if GitHub thinks you are spam, this happens when you send a lot of pull request/issue.
3. another problem that can occur is when you access on different devices in the account.

As you can see there are several problems that can occur with what you described. There may be other errors or issues that I'm not aware of, contact the GitHub support team to see how this can be resolved.

**Reply by ArtemKutsiy** [reply]
Where can I contact support? I wrote a ticket and have not had an answer there for 24 hours.

**Reply by ghost** [reply]
Hi ArtemKutsiy! There are 2 links to contact support: [GitHub Discussions](https://github.com/community/community/discussions) and [GitHub Support](https://support.github.com). It takes time for the GitHub team to respond, this is because there are multiple requests the team has/must address. In my case, there was after 3 days. I waited 3 days for them to respond. So what I'm saying is now you wait, although I don't know the time.

**Reply by sec1-info** [reply]
I'm waiting for over than 2 weeks and still no response

**Reply by sriharshaguthikonda** [reply]
i have been waiting for 70 days from last response! there are many issues with github..dont even want to start the discussion!

**Reply by Olympusxvn** [reply]
Help me my account flagged sir, i need account github for Hackathon, plz. i deleted all  clear. i will do care in future.

**Comment by yoogakim** [comment]
I have the same issue here

**Reply by shivani-bondstate** [reply]
I have same issue.

**Reply by cuong02n** [reply]
Did it solved?

**Comment by criatura2** [comment]
Hi, how can I unflag my account?

**Comment by Siddkamdar** [comment]
My account has been flagged. I would like to request a review for the same

**Comment by philipv99** [comment]
I haven't used GitHub for 4 years, I just reopened my account, and now I'm flagged. why is this?

**Comment by paypaxio** [comment]
hi there, we are creating our company's page! then we're flagged! why is that so? there was no other way. of course i am a developer and company is a different individual. they asked for their github account and we created it for them. what is flagged status about? this is a brand account! not spam or anything else!!!!

**Comment by QuickTradeSistemas** [comment]
Hello, Mr. or Ms.
I recently created this user in git
I don’t understand what happened to my account.
Would you tell me what to do now?
Thank you very much for your support.
I don’t know why my account has been flagged.

**Comment by BenFordTytherington** [comment]
My account also got flagged, seemingly by creating a gist, it just popped up at the top of the screen. I don't believe I have violated any rules so please review this. Thank you.

**Comment by Mitchelthepotato** [comment]
Hi my account had been flagged as well. Could someone review my account? Thanks a lot!

**Comment by SamaraEscudero** [comment]
I have the same message, but I don't have two accounts nor do I know the reason for the flag. How do I fix this?

**Comment by ronymolkanen** [comment]
Hi, my account is flagged atm. Can someone please review my account, thanks!!

**Comment by KingsandCatapults** [comment]
Hi my account had been flagged as well. Could someone review my account? Thanks a lot!

**Comment by morikee** [comment]
My too was erroneously flagged i only partake in gitcoin grant nothing else and i don't have two account only 1 account also this account very old not recent and i was flagged as spam kidly help me my ticket was open for almost a week now no reply

**Comment by dieu-bis** [comment]
Same issue for me.

**Comment by nitishsaxena16** [comment]
My account has been flagged as well, please help and remove the restrictions.

**Comment by CodingWale** [comment]
My account has been flagged. nothing wrong has been done. Kindly restore the acount status back to normal

**Comment by ossamamehmood** [comment]
Wonder to hear that GitHub flagged accounts too

**Reply by CaptainTee** [reply]
Mine has been resolved now

**Reply by ossamamehmood** [reply]
Great @CaptainTee

**Comment by morikee** [comment]
They refused to reinstate my account 😭

**Comment by hoainam0707** [comment]
My account has been flagged. I did nothing wrong

**Comment by stevenrobson** [comment]
This has now happened to me. I am a single owner LLC hobbyist on the side that published 2 free mobile apps. Is that grounds for getting flagged and removed from public view?

**Comment by JsCodeTee** [comment]
I have also experienced this issue that my account was flagged and removed from public view. I recently signed up, and don't have more than 1 account. What can be done to resolve this?

**Comment by yungsamd17** [comment]
My account has been flagged as well for some reason. Even though I didn't do anything to break the TOS.

**Reply by yungsamd17** [reply]
Got my account unflagged just a little over 24 hours, apparently it was:

> _"Spam detecting systems highlighted my account to be needed to be manually reviewed"_

**Comment by Generalsimus** [comment]
Hi my account had been flagged as well. Could someone review my account?😐 Thanks a lot!

**Reply by ghost** [reply]
gitlab

**Comment by speech2sing** [comment]
My account has been flagged as well for some reason, I just signed up 2 weeks ago and I complete the 2FA setting.

**Comment by ghost** [comment]
they are crazy that's perfectly illegal what they are doing. someone report you but you can't defend yourself. i didn't commit a murder neither insulted someone. and even in this case, is it harassment one word. I told pervert. you are insane there. but totally

**Comment by shashank-0-0** [comment]
Even my account shashank-0-0 is flagged though i have only one account and other one is my work email .They have not responded for 2 weeks now

**Comment by shashank-shetty-0-0** [comment]
Even my personal account [shashank-0-0](https://github.com/shashank-0-0) is flagged for no reason and they have not been responding for like 2 weeks now.

**Comment by jaydipkanpariya** [comment]
My account flagged

How to remove flagged from my account?

**Comment by jaydipkanpariya** [comment]
How to remove this flage from my account?

On Sat, Apr 26, 2025 at 1:02 AM Sayori ***@***.***> wrote:

> I'm waiting for over than 2 weeks and still no response
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/30093#discussioncomment-12950684>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQUQ6FMCXIEIMJOHKDNCBFD23KEUXAVCNFSM6AAAAAB34LR4VOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOJVGA3DQNA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
*-------------------------*

*Kanpariya Jaydip G.*

**Comment by Adityayadav776** [comment]
my account is flagged cant login from third party apps?
I was trying to push a folder of my project using git command line. 
please help
![Untitled](https://github.com/user-attachments/assets/0425a59f-034c-47da-8c1f-3986637aa71c)

**Reply by zentheriun** [reply]
Yes — that error means exactly what it says: **your GitHub account is flagged**, and as a result, **you can’t authorize or use third-party applications**, including pushing from the Git command line, CI tools, GitHub Desktop, or any API-based access.

---

### 💥 Why this happens

When an account is **flagged**, GitHub restricts it to prevent potential abuse. This affects:

* Git command-line access (`git push`, `git clone`, etc.)
* OAuth/third-party apps
* API tokens
* Webhooks and automation

Even if your login works, GitHub blocks external tool access **until your account is reviewed**.

---

### ✅ How to fix it

You **must contact GitHub Support** to resolve this. Here's what to do:

1. Go to: [https://support.github.com/contact](https://support.github.com/contact)
2. Choose **"Account flagged"** as the issue type.
3. Use this message (or similar):

> Hello GitHub Support,
> I’ve received the message “This account is flagged and cannot authorize a third-party application.”
> I’m trying to push a project using the Git CLI, but it’s blocked.
> I believe this may be a mistake — I’d appreciate it if you could review and clarify why my account is flagged, and if it can be restored.
> Thank you very much!

---

### 🧠 Tip

* **Be polite and clear** in your message.
* Don’t try to create a second account — GitHub might block it too.
* If you use GitHub for school/work, mention that.

---

**Comment by GoddeyUwamari** [comment]
My account has been flagged and it has been one year i have started submitting requests for reinstatement and yet, no response. What do i do?

**Comment by nogeek-net** [comment]
My account has been flagged. I did nothing wrong

**Reply by swh922** [reply]
My account had been flagged as well, This account is flagged, and therefore cannot authorize a third party application. I have submitted a ticket (Ticket 3481323) and hope that the community management team can see my message!

**Reply by JRH89** [reply]
Same please help

**Comment by aliveevie** [comment]
Please GitHub team, help to unflagged my account. I did no wrong 😭😭😭

**Comment by aliveevie** [comment]
@github team,
I'm building something very important and suddenly just flagged my account, please unflagged me.

**Comment by aliveevie** [comment]
It's unfortunate to see that my account was flagged and became invisible to public and also all my third party apps that I signed to GitHub has been revoked and I can't longer sign up again.
Please what is happening to you GitHub.
I raised a ticket and Nobody responded to it.
It's really difficult time to me.
I'm just disappointed with you All of Github

**Comment by zentheriun** [comment]
Hi @30093,

The message **“Your account has been flagged”** means that GitHub has marked your account due to some unusual or potentially policy-violating activity.

---

### ❓ What does "flagged" mean?

A **flagged account** is one that GitHub has temporarily restricted because it may have:

* Posted spam or promotional content (even unintentionally),
* Used bots or automated tools aggressively (e.g., mass following/forking),
* Violated GitHub's [[Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service)](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service) or [[Community Guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines)](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines),
* Or simply triggered the system by mistake.

When your account is flagged:

* Your profile may become invisible to others,
* You may lose access to features like opening issues, commenting, creating repos, etc.,
* GitHub limits your ability to interact until the situation is reviewed.

---

### ✅ How to fix it

You’ll need to **contact GitHub Support directly** — only they can investigate and remove the flag.
You can do that here:

👉 [https://support.github.com/contact](https://support.github.com/contact)

#### 📨 Here's a message you can send:

> Hello GitHub Support,
> I noticed my account was flagged with the message “Your account has been flagged.”
> I’d like to understand why this happened and if it can be reviewed.
> As far as I know, I’ve followed the community rules.
> If there was a mistake or misunderstanding, I’m happy to fix it.
> Thank you!

GitHub typically responds within a few business days.

---

🔒 **Note**: Regular users and community members can’t remove this flag — only GitHub’s support team can help.

---

**Comment by sec1-info** [comment]
I’ve done all of that but support take 2~3 months to respond and yet they
didn’t resolve the problem yet.
I’ve missed a lot of work because of this issue

**Reply by zentheriun** [reply]
I'm really sorry to hear that — I completely understand how frustrating and stressful this must be, especially if it's affecting your work.

Unfortunately, once an account is flagged, only GitHub Support can remove the restriction, and I agree, the response times can be very slow lately (sometimes several weeks or even months, as you've experienced).

**Comment by zeyi2** [comment]
My account was flagged one week ago, I opened a ticket and it was closed directly without a response. I don't know what I did wrong, and I missed a lot of deadlines because of this issue. Please, github support, help.

**Reply by theDreamer911** [reply]
it still happended on me, if you read my comment it means the flag already taken and my account back to normal, I hope we could find solution from this

**Comment by IIIdefconIII** [comment]
Same here and i cant find realible support for it...

**Comment by sound4c** [comment]
My account is flagged, how do I get help?

**Reply by theDreamer911** [reply]
I need help too

**Comment by eskayML** [comment]
My account has been flagged 

i have been using it without violating any TOS for the past 7 years

**Comment by ptanh12** [comment]
My account was flagged right after creation. My company required me to create a new github account using the company email.
<img width="1272" height="576" alt="image" src="https://github.com/user-attachments/assets/bce813bb-9de8-4419-a733-58af2a9ea64d" />

**Comment by kauepsilva** [comment]
Hello, my account appears to be flagged by an automated system. I am receiving the error "This account is flagged, and therefore cannot authorize a third party application" when trying to connect external services. My account is legitimate and I use it only for development. Could you please review my case and remove this restriction? Thank you.

**Reply by hand-agent** [reply]
me too, I created this account with no intention for using it for so long, if you already found the solution, please tell me, thankyou

**Reply by theDreamer911** [reply]
this is my account that had been flagged by GitHub, I hope their teams doesn't rely on bot detection only, someone needs to fix this problem

**Reply by IIIdefconIII** [reply]
i just created a new account which was lame to have to do

**Comment by AsimHateez** [comment]
Hey there,
   So i did not find the solution but i can tell you the exact process i
followed. SO this is how it went .
I directly email the GitHub support and they respond me like some of my
repositories have the bugs/virus that is causing the account to be
restricted for the third party connections and the github backend flagged
the account cause of this. So they asked me to delete those repos and email
them about the delection is complete so they would  take the
further needful actions after couple of days it was done now i am using it
smoothly.

**Reply by hand-agent** [reply]
what is their email address

**Comment by AsimHateez** [comment]
I think is there email:
***@***.***
